packages/squidex/schema/schemas/users.json

```diff
@@ -280,26 +280,6 @@
           "isHalfWidth": true
         }
       },
-      {
```

Comment:
I assume this means we lose the data in production. Have we confirmed this with bea?

Reply:
![image](https://user-images.githubusercontent.com/11645956/126146363-8424b69a-9596-4d61-ae6c-708090222ce6.png)
I think so - i mean the ticket clearly says to verify that the data was migrated to city/country and its crossed out now, but ill speak with B just in case

Reply:
although - we do weekly backups so it wont be gone gone